test/integration/settings_test.rb

```diff
@@ -111,6 +111,49 @@ def mfa_key
     assert page.has_content? "You have not yet enabled multifactor authentication."
   end
 
+  test "Clicking MFA continue button without copying recovery codes creates confirm popup" do
+    sign_in
+    visit edit_settings_path
+    click_button "Register a new device"
+
+    totp = ROTP::TOTP.new(mfa_key)
+    page.fill_in "otp", with: totp.now
+    click_button "Enable"
+    check "ack"
+
+    page.dismiss_confirm do
+      click_button "Continue"
```

Comment:
nit: it isn't too clear the difference between the default and the custom popup in these tests. Perhaps also verifying the message in this popup would help.